web/packages/teleterm/src/services/tshd/createClient.ts

```diff
@@ -295,21 +295,8 @@ export function createTshdClient(
     },
 
     async createAccessRequest(params: types.CreateAccessRequestParams) {
-      const req = api.CreateAccessRequestRequest.create({
```

Comment:
On v14, `params` have to be converted to a class. This means you'll have to call setters for the fields added in this PR, just like it's done for other fields already.

https://github.com/gravitational/teleport/blob/fe62f5b9089b63a744d47f8a51722f5404342542/web/packages/teleterm/src/services/tshd/createClient.ts#L240-L254